.clang-format

```diff
@@ -100,7 +100,7 @@ SpaceBeforeCtorInitializerColon: false
 SpaceBeforeInheritanceColon: false
 SpaceBeforeParens: ControlStatementsExceptControlMacros
 SpaceBeforeRangeBasedForLoopColon: false
-SpaceBeforeCpp11BracedList: false
+SpaceBeforeCpp11BracedList: true
```

Comment:
Changing this affects a bunch of other things. I think it would be better to switch to the apt.llvm.org version on your laptop.